### PR TITLE
Run all tests at once.

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

**Comment by Nyholm** [comment]
There is something wrong with the ssh protocol to geocoder.ca. A part from that, this is fine. I'll update later.

**Comment by willdurand** [comment]
Yep, that makes perfectly sense now that all providers have been fixed!

**Comment by vicchi** [comment]
> There is something wrong with the ssh protocol to geocoder.ca

SSL rather than SSH?

**Comment by Nyholm** [comment]
Of course. Sorry.

**Comment by Nyholm** [comment]
The failing tests is because of the OIORest provider. They are out of service. See #43

File: src/Geocoder/Provider/What3wordsProvider.php [hunk 1]
```diff
@@ -56,7 +56,8 @@ protected function executeQuery($query)
     {
         $content = $this->getAdapter()->get($query)->getBody();
 
-        if (null === $data = json_decode($content, true)) {
+        $data = json_decode($content, true);
+        if ($data === null || empty($data)) {
```
Comment (Nyholm) [inline src/Geocoder/Provider/What3wordsProvider.php]: @vicchi It seems that I was wrong in both my comments on #42 

Would you mind giving this PR a review and merge it if you like it

**Comment by vicchi** [comment]
+1/LGTM ... will merge

**Pull request merged**